maven/impl-maven-embedded/src/test/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/ConfigurationStageTestCase.java

```diff
@@ -81,12 +80,13 @@ public void runTest() {
         assertEquals(shellEnvironments, invocationRequest.getShellEnvironments());
         assertEquals(threads, invocationRequest.getThreads());
         assertEquals(toolChainsFile, invocationRequest.getToolchainsFile());
+        assertEquals(globalToolChainsFile, invocationRequest.getGlobalToolchainsFile());
         assertEquals(userSettingFile, invocationRequest.getUserSettingsFile());
-        assertEquals(true, invocationRequest.isActivatedReactor());
+        assertEquals(builderId, invocationRequest.getBuilder());
```

Comment:
Also maybe worth to split this test to multiple ones if we test more than one aspect of the build here?